WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostListFragment.java

```diff
@@ -423,22 +434,19 @@ public void onCreate(Bundle savedInstanceState) {
             initSubFilterViewModel();
         }
 
-        mViewModel.getShouldCollapseToolbar().observe(this, collapse -> {
-            if (collapse) {
-                mRecyclerView.setToolbarScrollFlags(AppBarLayout.LayoutParams.SCROLL_FLAG_SCROLL
-                                                    | AppBarLayout.LayoutParams.SCROLL_FLAG_ENTER_ALWAYS);
-            } else {
-                mRecyclerView.setToolbarScrollFlags(0);
-            }
-        });
-
         handleReblogStateChanges();
 
-        mViewModel.start(!mIsTopLevel, mReaderViewModel);
+        mViewModel.start(mReaderViewModel);
 
         if (isFollowingScreen()) {
             mSubFilterViewModel.onUserComesToReader();
         }
+
+        if (getPostListType() == ReaderPostListType.SEARCH_RESULTS) {
+            mRecyclerView.showAppBarLayout();
+            mSearchMenuItem.expandActionView();
+            mRecyclerView.setToolbarScrollFlags(0);
```

Comment:
The Toolbar shouldn't scroll with the content of the search suggestions and search results -> the search input is in the toolbar and it should be always visible so the user can easily get back to it (this is the same behavior as in develop).